### PR TITLE
GSHN0W - Dummy sensor implementation

### PR DESCRIPTION
**Pull request by nnorbi3**

### CODE REVIEW

File: src/AutomatedCar/SystemComponents/Sensors/DummySensor.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        public override void Process()
+        {
+            Circle circle = (Circle)World.Instance.WorldObjects[0];
+            AutomatedCar car = World.Instance.ControlledCar;
+            int dX = Math.Abs(circle.X - car.X);
+            int dY = Math.Abs(circle.Y - car.Y);
+            dummyPacket.DistanceX = dX;
+            dummyPacket.DistanceY = dY;
+            virtualFunctionBus.DummyPacket = dummyPacket;
```
Comment (torokt21) [inline src/AutomatedCar/SystemComponents/Sensors/DummySensor.cs]: Unnecessary line. DummyPacket is a reference type, setting it in the constructor is enough.

**Review by torokt21 — CHANGES_REQUESTED** [review]
Good solution, but one minor issue

File: src/AutomatedCar/SystemComponents/Sensors/DummySensor.cs (hunk 1 above)
Comment (nnorbi3) [reply src/AutomatedCar/SystemComponents/Sensors/DummySensor.cs]: Valid point, thanks!

**Review by torokt21 — APPROVED** [review]

**Comment by jozdavab** [comment]
Better late than never

**Pull request closed without merging**